test/testvm.py

```diff
@@ -776,6 +834,8 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
             "-kernel", self._image_kernel,
             "-initrd", self._image_initrd
         ]
+        if self.init_image:
+            cmd += ["-cdrom", self.init_image]
```

Comment:
> Does every boot of Atomic need the init ISO? Or do we only need it during vm-create?

I guess I am confused.  Looks like every run does indeed need that ISO image, and f-a-22.bootstrap does indeed produce the expected qcow2.

Can we give the ISO a special name, like "cockpit-fedora-atomic-22-x86_64.iso", and if such a file exists, testvm will put it into the cdrom.  That way we can avoid mentioning atomic in testvm.

Also, vm-save and vm-download should handle the ISO.